npm-debug.log

```diff
@@ -0,0 +1,22 @@
+0 info it worked if it ends with ok
+1 verbose cli [ 'C:\\Program Files\\nodejs\\node.exe',
+1 verbose cli   'C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js',
+1 verbose cli   'start' ]
+2 info using npm@3.10.10
+3 info using node@v6.11.5
+4 verbose stack Error: ENOENT: no such file or directory, open 'C:\Users\Lava\Documents\Perkpack\package.json'
```

Comment:
I dont know if we should be merging this in if its pointing to the C drive of my pc.